pyproject.toml

```diff
@@ -61,7 +61,6 @@ python = ">=3.7.0,<3.12"
 argcomplete = "^2.0.0"
 humanize = "^4.4.0"
 pillow = "^9.4.0"
-pyyaml = ">=5.1"
```

Comment:
This was breaking for me for some reason unless pyyaml was removed and then added back. Unsure why.